src/wallet/output_alias.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __WALLET_OUTPUT_ALIAS_H__
+#define __WALLET_OUTPUT_ALIAS_H__
+
+#include "wallet/wallet.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Send transaction which creates a new alias output
+ *
+ * @param[in] w A wallet instance
+ * @param[in] sender_addr The sender address
+ * @param[in] sender_keypair The sender private key
+ * @param[in] send_amount The amount to transfer
+ * @param[in] state_ctrl_addr The receiver address
+ * @param[in] govern_addr The receiver address
+ * @param[out] alias_addr The response of the transfer
+ * @param[out] msg_res The response of the transfer
+ *
+ * @return int 0 on success
+ */
+int wallet_alias_create_send(iota_wallet_t* w, address_t* sender_addr, ed25519_keypair_t* sender_keypair,
+                             uint64_t const send_amount, address_t* state_ctrl_addr, address_t* govern_addr,
+                             address_t* alias_addr, res_send_message_t* msg_res);
```

Comment:
the wallet is able to derive the sender address and keypair from BIP44's path inside this method, it also reduces the coding which the user doesn't need to call `get_address_and_keypair`, like in the `examples/wallet/create_alias_output.c` 
maybe rename to `wallet_alias_output_create`?
```suggestion
int wallet_alias_create_send(iota_wallet_t* w, bool sender_change, uint32_t sender_index,
                             uint64_t const send_amount, address_t* state_ctrl_addr, address_t* govern_addr,
                             address_t* alias_addr, res_send_message_t* msg_res);
```

Please add a TODO comment about the optional Feature blocks. let's revisit it when needed.
```
// TODO: the alias output should be able to set optional feature blocks such as Sender/Metadata
```

Reply:
Thank you for all code suggestions and improvements.